### PR TITLE
ready for review

### PR DESCRIPTION
**Pull request by Nathan-reitan**
SGR-React-add-grade

### CODE REVIEW

File: server/public/style.css [hunk 1]
```diff
@@ -2,3 +2,31 @@
   display: flex;
   justify-content: space-between;
 }
+
+.container{
+  display: flex;
+  flex-wrap: wrap;
+
+}
```
Comment (thebearingedge) [inline server/public/style.css:10]: Bootstrap already has a `container` class. Perhaps some of their `flex` helper classes are what you're after.

File: server/public/style.css [hunk 2]
```diff
@@ -2,3 +2,31 @@
...
+
+button{
+  margin: 15px;
+}
+
+input{
+  position: relative;
+  left: 10px;
+}
```
Comment (thebearingedge) [inline server/public/style.css:32]: Try margin. Bootstrap has spacing helper classes.

File: server/public/style.css [hunk 3]
```diff
@@ -2,3 +2,31 @@
...
+
+label>div{
+  height: 28px;
+  width: 28px;
+}
+
+button{
+  margin: 15px;
+}
```
Comment (thebearingedge) [inline server/public/style.css:27]: Use helper classes if you can.

File: server/public/style.css [hunk 4]
```diff
@@ -2,3 +2,31 @@
...
+
+}
+
+.form{
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  font-size: 1.5em;
+}
```
Comment (thebearingedge) [inline server/public/style.css:18]: Bootstrap has flex helpers.

**Comment by Nathan-reitan** [comment]
image

**Pull request merged**